manifests/init.pp

```diff
@@ -49,6 +49,7 @@
   $service_name       = 'mcollective',
   $server_package     = 'mcollective',
   $ruby_stomp_package = 'ruby-stomp',
+  $ruby_interpreter   = '/usr/bin/env ruby',
```

Comment:
Is this pe compatible? There are some other PRs floating around that munge intepreter to be pe safe